playwright/mockData/Memberships.ts

```diff
@@ -0,0 +1,16 @@
+import KPD from './orgs/KPD';
+import RosaLuxemburg from './users/RosaLuxemburg';
+import { ZetkinMembership } from '../../src/types/zetkin';
+
+const Memberships: ZetkinMembership[] = [
+  {
+    organization: KPD,
+    profile: {
+      id: RosaLuxemburg.id,
+      name: RosaLuxemburg.first_name + ' ' + RosaLuxemburg.last_name,
+    },
+    role: 'admin',
+  },
+];
+
+export default Memberships;
```

Comment:
This would probably be better inside the KPD section. Because this is "memberships" for KPD members